src/services/QueueManager.ts

```diff
@@ -135,10 +121,10 @@ export default class QueueManager {
 				logger.error('[critical] Unable to connect to the Redis server - Notification Service is going to exit!');
 				process.exit(1);
 			}
-			logger.error('Unable to connect to the Redis server ..retry', { times });
+			logger.error('Unable to connect to Redis server, retrying...', { times });
```

Comment:
what happens if a job is taken... do we have a guarantee it reaches the redis store or returns an error?

Reply:
I updated connection error handling in [3b1ff2d](https://github.com/hpi-schul-cloud/node-notification-service/pull/279/commits/3b1ff2d50ffb9bf0a14b70ec8ede61f1f5469d49):
- initially we require a successful redis connection, otherwise the service will exit
- when the redis connection is lost while the service is running, ioredis will try to reconnect in specific interval (currently 5s)
- this results in HTTP 503 responses (Service temporarily unavailable) until the redis connection is available again